global-news-app/src/Hooks/SearchContext/queryHooks/useGetArticlesByLocation.ts

```diff
@@ -0,0 +1,19 @@
+import { getArticlesByUserLocation } from "services/api-services";
+import { useQuery } from "react-query";
+import { QueryKeys } from "Types/enums";
+
+const useGetArticlesByLocation = (countryCode: string = "") => {
+  const { data, isSuccess, isLoading } = useQuery(
+    [QueryKeys.UserLoaction, countryCode],
+    () => getArticlesByUserLocation(countryCode),
+    {
+      // enabled: Boolean(countryCode),
```

Comment:
remove the comments if the code is not used
in case you need it in future add the comment with the TODO
      `// TODO: some comment`